packages/ckeditor5-table/tests/table-integration.js

```diff
@@ -187,13 +187,41 @@ describe( 'Table feature – integration', () => {
 		} );
 
 		it( 'should not make the Model#hasContent() method return "true" when an empty table cell is selected', () => {
-			setModelData( editor.model, '<table>' +
-				'<tableRow>' +
-					'[<tableCell><paragraph></paragraph></tableCell>]' +
-				'</tableRow>' +
-			'</table>' );
+			setModelData( editor.model, (
+				'<table>' +
+					'<tableRow>' +
+						'[<tableCell><paragraph></paragraph></tableCell>]' +
+					'</tableRow>' +
+				'</table>'
+			) );
 
 			expect( editor.model.hasContent( editor.model.document.selection.getFirstRange() ) ).to.be.false;
 		} );
 	} );
 } );
+
+describe( 'Table feature – integration #2', () => {
```

Comment:
Not the best name for describing the test case. What is `#2`? I'd describe what it integrates.